applications/FluidDynamicsApplication/tests/compressible_navier_stokes_symbolic_generator_formulation_test.py

```diff
@@ -0,0 +1,115 @@
+import os
+
+import KratosMultiphysics
+import KratosMultiphysics.KratosUnittest as KratosUnitTest
+
+from KratosMultiphysics.FluidDynamicsApplication.symbolic_generation.compressible_navier_stokes \
```

Comment:
this uses `sympy` which is not part of the python standard library
Hence if someone (like me :D) that does not have sympy installed, then this test (and the entire test script) crashes 

Hence I recommend protecting this with a check in `test_FluidDynamicsApplication.py`
(see e.g. how `numpy` is handled in the [CoSim-tests](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/CoSimulationApplication/tests/test_CoSimulationApplication.py))

Reply:
Good point